R/calculate_within_batch_ari.R

```diff
@@ -26,6 +29,7 @@ calculate_within_batch_ari <- function(individual_sce_list,
                                        pc_names,
                                        merged_sce,
                                        batch_column = "library_id",
+                                       barcode_column = "barcode",
```

Comment:
Will we mostly want to use `barcode` here instead of `cell_id`?

My reason for asking is that I checked the `barcode` column of our example integrated SCE object and it seems to be filled with NAs:

<img width="799" alt="Screen Shot 2023-08-09 at 11 02 05 AM" src="https://github.com/AlexsLemonade/scpcaTools/assets/43576623/d6e02fc2-3ab9-4dca-b0fc-1ec7a0f656fb">

Whereas, `cell_id` gives us:

<img width="744" alt="Screen Shot 2023-08-09 at 11 03 19 AM" src="https://github.com/AlexsLemonade/scpcaTools/assets/43576623/881e93a6-8f3b-4da3-a4fd-0d382bf01343">

Reply:
Good catch! We have an argument in `merge_sce_list()` that sets the default cell column to be `cell_id`. I went ahead and changed the argument name and default to match what we already use there: 
https://github.com/AlexsLemonade/scpcaTools/blob/7982096108d031510eb925d1d6d600caf91a4c7f/R/merge_sce_list.R#L59